### PR TITLE
Added info about new Ansible settings

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: plugins/foreman_ansible/1.x/index.md [hunk 1]
```diff
@@ -90,6 +90,10 @@ If the Foreman setting 'create_new_host_when_facts_are_uploaded' (Puppet tab) is
 
 Similarly, the Foreman setting 'ignore_puppet_facts_for_provisioning' (Provisioning tab) is set to false, facts related to interfaces will update the interfaces of $HOSTNAME in Foreman.
 
+There are several Ansible options you can configure under Administer -> Settings in case you need to configure the Ansible port, user, etcetera. You can override these options on any host via Host parameters, Host group parameters or Global parameters by setting the attributes `ansible_port`, `ansible_ssh_pass` or `ansible_user`.
+
+![ansible settings](static/images/plugins/foreman_ansible/settings.png)
```
Comment (tbrisker) [inline plugins/foreman_ansible/1.x/index.md:95]: I think you forgot to add the image to the commit.

Comment (dLobatog) [reply plugins/foreman_ansible/1.x/index.md:95]: Well that's certainly important!

**Comment by dLobatog** [comment]
Added the settings png file - thanks @tbrisker for taking a look!

**Comment by mmoll** [comment]
merged with small fixups (single quote vs. backtick), gracias @dLobatog!

**Pull request closed without merging**